system-variables.md: update tidb_ddl_enable_fast_reorg (#11448)
pingcap/docs: system-variables.md

- Scope: GLOBAL
- Persists to cluster: Yes
- Type: Boolean
- Default value: `ON`

> - This variable controls whether to enable the acceleration of `ADD INDEX` and `CREATE INDEX` to improve the speed of backfilling when creating an index. The performance improvement is especially significant for tables with a large amount of data.

Reviewer: ```suggestion
- This variable controls whether to enable the acceleration of `ADD INDEX` and `CREATE INDEX` to improve the speed of backfilling for index creation. Setting this variable value to `ON` can bring performance improvement for index creation on tables with a large amount of data. 
```

> - This variable controls whether to enable the acceleration of `ADD INDEX` and `CREATE INDEX` DDl operations to improve the speed of backfilling when creating an index. If this variable is enabled, TiDB uses a more effective way to create an index especially for large tables.

Reviewer 2: ```suggestion
- This variable controls whether to enable the acceleration of `ADD INDEX` and `CREATE INDEX` to improve the speed of backfilling when creating an index. The performance improvement effect is especially obvious under tables with a large amount of data.
```

> - To verify whether a completed `ADD INDEX` operation is accelerated, you can execute the [`ADMIN SHOW DDL JOBS`](/sql-statements/sql-statement-admin-show-ddl.md#admin-show-ddl-jobs) statement to see if `ingest` is displayed in the `JOB_TYPE` column.

Reviewer: ```suggestion
- To verify whether a completed `ADD INDEX` operation is accelerated, you can execute the [`ADMIN SHOW DDL JOBS`](/sql-statements/sql-statement-admin-show-ddl.md#admin-show-ddl-jobs) statement to see whether `ingest` is displayed in the `JOB_TYPE` column.
```
